File: ComposerCMS.Web/ClientApp/src/app/models/ExternalResource.ts
export class ExternalResource {
  id: string;
  externalResourcePackageId: string;
  name: string;
  extension: string;
  path: string;
  order: string;
  dateAdded: Date;
  dateLastUpdated: Date;
  userIdAdded: string;
  userIdUpdated: string;

  // TODO: Convert this to a model.
  userName: string;
}

File: ComposerCMS.Core/CoreSystem/DAL/Entity/ExternalResource.cs
using ComposerCMS.Core.Interface;
using System;

namespace ComposerCMS.Core.Entity
{
    public class ExternalResource : IEntityTracking
    {
        public Guid ID { get; set; }

        public Guid? ExternalPackageID { get; set; }

        /// <summary>
        /// File name
        /// </summary>
        public string Name { get; set; }

        public string Extension { get; set; }

        /// <summary>
        /// wwwwroot/composer-cms/[css|js]
        /// </summary>
        public string Path { get; set; }

        public int Order { get; set; }

        public DateTime DateAdded { get; set; }
        public DateTime DateLastUpdated { get; set; }
        public Guid UserIDAdded { get; set; }
        public Guid UserIDLastUpdated { get; set; }
    }
}